### PR TITLE
Drop database if exists

### PR DESCRIPTION
**Pull request by JacobMcAnly**
In order to reset our database we want to drop the database if it exists. This will allow us to recreate it

### CODE REVIEW

File: database/create_prs.sql [hunk 1]
```diff
@@ -1,7 +1,8 @@
 USE master;
 GO
 
--- Needs a drop if exists so that you can recreate this by just executing the script
```
Comment (kaitannmatthey-max) [inline database/create_prs.sql:4]: Put this line back in

**Review by kaitannmatthey-max — CHANGES_REQUESTED** [review]

File: database/create_prs.sql (hunk 1 above)
Comment (JacobMcAnly) [reply database/create_prs.sql:4]: I don't believe we need this line

Comment (kaitannmatthey-max) [reply database/create_prs.sql:4]: You know what. You're right. Thanks!

**Review by kaitannmatthey-max — APPROVED** [review]
Looks good to me!

**Pull request merged**